docs/source/releasenotes.rst

```diff
@@ -227,6 +227,17 @@ Releases on 01 Dec 2021
 11.0.0
 ------
 
+- Migration guide: Given this major upgrade, the ``Load Work Item ...`` keywords got
```

Comment:
This is how it's rendered:
![Uploading Screenshot 2021-12-16 at 12.33.07.png…]()